### PR TITLE
Akashic Tome Fix

### PR DESCRIPTION
**Pull request by Kanzaji**
So that is my fault ^^" I completly forgot to check if it actually works after import, and well no. It is causing crashes ( #224 ) so it is time to fix it!
I'm sorry about that, mistakes can happend 😅

For now it is a draft, when fix will be finished I will mark it as "Ready for Review" :D 

Note: I'm talking about quest one, it is marked as WIP but ig people are still taking it.

### CODE REVIEW

**Comment by Kanzaji** [comment]
Okay so Akashic Tome on 1.18.2 is a bit of a mess:

obraz

However it now works fine :D (and this is only visual bug)

I needed to remove 2 manuals (Tetra and Alexs Mobs) as those weren't able to be "merged" in crafting, and removed everything else that isn't in E8, and added few more.

Anyhow fixed!

**Comment by Kanzaji** [comment]
Okay I did realize you can in config add those manuals that aren't possible to be combined by default. Back to draft it is!

File: config/ftbquests/quests/chapters/getting_started.snbt [hunk 1]
```diff
@@ -170,586 +170,432 @@
 				""
 				"●Craft the Tome with a book to merge them once more. "
 				""
+				"Note: Akashic Tome is a bit broken, and book is covering half of the manuals. This is only visual bug, so book is 100% functional!"
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/getting_started.snbt:173]: Quite a few typos here, I'd rather you remove this section tbh.

**Review by NielsPilgaard — CHANGES_REQUESTED** [review]

**Comment by NielsPilgaard** [comment]
Also, thanks for making the fix :D

File: config/ftbquests/quests/chapters/getting_started.snbt (hunk 1 above)
Comment (Kanzaji) [reply config/ftbquests/quests/chapters/getting_started.snbt:173]: Ah alrighty. (And sorry for typos, was doing in that time few things at once 😅) Gonna remove it then.

**Comment by Kanzaji** [comment]
Alrighty, deleted that part about bug, any other changes you would like to have 😄 ?

**Comment by NielsPilgaard** [comment]
> Alrighty, deleted that part about bug, any other changes you would like to have 😄 ?

I think it's good 👍 there's a section about JER (which we don't have) in the Modded for Dummies book, but I can live with it 😄 Kinda hard to review the Akashic Tome change tbh.

**Review by NielsPilgaard — APPROVED** [review]

**Comment by Kanzaji** [comment]
I will take a note about JER and remove it in Quest Rework then 😅 As making PR for that change is I think not necessary.
And the biggest problem with Akashic Tome was the fact it was crashing the game, what it doesn't atm and that was the point!

**Pull request merged**